msm/src/serialization/mod.rs

```diff
@@ -8,24 +12,30 @@ pub mod witness;
 /// The number of intermediate limbs of 4 bits required for the circuit
 pub const N_INTERMEDIATE_LIMBS: usize = 20;
 
-#[derive(Clone, Copy, Debug, Hash, Eq, PartialEq, Ord, PartialOrd)]
-pub enum LookupTable {
+#[derive(Clone, Copy, Debug, Hash, Eq, PartialEq, Ord, PartialOrd, EnumIter)]
+pub enum LookupTable<Ff> {
     RangeCheck15,
     RangeCheck4,
+    RangeCheck4Abs,
+    RangeCheckFfHighest(PhantomData<Ff>),
```

Comment:
Can you add a comment describing the goal of the table, please?
Nit: I would mention what `Ff` means in a comment.